app/src/main/java/com/example/pokemonultimate/ui/screens/user/UserScreen.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.pokemonultimate.ui.screens.user
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+
+@Composable
+fun UserScreen(){
+        Text("page user")
+}
```

Comment:
Event if it's just for test please respect indentation

```suggestion
@Composable
fun UserScreen() {
    Text("page user")
}
```